public/apps/configuration/panels/role-edit.tsx

```diff
@@ -60,29 +124,55 @@ export function RoleEdit(props: RoleEditDeps) {
       </EuiPageHeader>
       <PanelWithHeader headerText="Name">
         <EuiForm>
-          <EuiFormRow
-            label={
-              <EuiText size="xs" color="subdued">
-                <EuiTitle size="xxs">
-                  <h4>Name</h4>
-                </EuiTitle>
-                Specify a descriptive and unique role name. You cannot edit the name onece the role
-                is created.
-              </EuiText>
-            }
+          <FormRow
+            headerText="Name"
+            headerSubText="Specify a descriptive and unique role name. You cannot edit the name onece the role is created."
             helpText="The Role name must contain from m to n characters. Valid characters are 
             lowercase a-z, 0-9 and (-) hyphen."
```

Comment:
are you planing to add ux validation? or will ignore the illegal character directly while user typing.

Reply:
The validation will come later. The length and valid character set are TBD. Should include `_` since the some default roles do have `_` in their names.